packages/terra-application-layout/src/header/_ApplicationHeader.jsx

```diff
@@ -1,7 +1,7 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 import classNames from 'classnames/bind';
-import { injectIntl, intlShape } from 'react-intl';
+import { injectIntl, intlShape } from 'terra-base';
```

Comment:
We've run into some issues with pulling in react-intl modules this way from terra-base. More info on this issue here: https://github.com/cerner/terra-core/issues/1982
For now, let's leave this as pulling from react-intl.